1_space_odyssey/js/simen-9000/app.js

```diff
@@ -125,4 +125,70 @@
 }
 else {
     console.log("JetPack NOT enabled, please enable jetPack before doing a space walk");
-}
\ No newline at end of file
+}
+/*
+ 3. METEOR SHOWER::::::
+    On your way to Jupiter you encounter a massive meteor shower that could damage the ship, you need to redirect the ship to a new safe location.
+    a) Using a for loop iterate through the responses array and choose the appropriate response to get out of danger.
+      create an if condition in your loop to print only 1 response using a comparison operator.
+*/
+console.log(
+    "\u{1F327} METEOR SHOWER::::::"
+  );
+  
+  var responses = [
+    "Call basecamp and ask for assistance, but they are far away",
+    "Do nothing, just push through",
+    "initiate core thrusters, punch 180degrees into dash-nav and warp away from danger",
+    "ask humanoid-9000 to fly out there and shield the ship from the debris, this action could harm the humanoid"
+  ];
+  
+  
+  // write your code here, use the console log beneath
+  // console.log("\u{1F6F8} Swoosh -> -> Ship is steered to safety!");
+  
+  for (var i = 0; i < responses.length; i++) {
+    if (responses[i] === "initiate core thrusters, punch 180degrees into dash-nav and warp away from danger") {
+        console.log("\u{1F6F8} Swoosh -> -> Ship is steered to safety!");
+    }
+}
+  
+  /*
+   4. ALIENS SPACE CRAFT ENCROACHING::::::
+      Alien craft approaches its an enemy for sure...you need to shoot it down!!
+      a) convert the pseudoscope to real code
+         create the required variables below and fill in the correct values and dataTypes to make
+         the if statements pass so that photonLaser fires pew pew pew.
+      b)
+  */
+  console.log(
+    "\u{1F47E} ALIENS SPACE CRAFT ENCROACHING::::::"
+  );
+  
+  // a.
+  var enemyCraft = true;
+  var photonLaser = true;
+  var photonLaserEnabled = true;
+  var bullets = 100;
+
+  if (enemyCraft && photonLaser && photonLaserEnabled); {
+      if (bullets >= 100) {
+          for (var i = 0; i < 100; i++){ console.log("\u{2708} \u{1F525} pew pew pew !! you got them!!");
+      }
+          } else {
```

Comment:
please clean code indent correctly .
https://github.com/mannuelf/clean-code-javascript
have a look at all the if statement in that doc ;-)